src/ziggurat/server.clj

```diff
@@ -19,10 +34,13 @@
                              :min-threads          thread-count
                              :max-threads          thread-count
                              :join?                false
-                             :send-server-version? false})))
+                             :send-server-version? false
+                             :configurator         configure-jetty})))
 
 (defn- stop [^Server server]
-  (.stop server)
+  (try
+    (.stop server)
+    (catch TimeoutException _ (log/info "Server Stop timed out")))
```

Comment:
Can we making this log statemetn little mote informative possibly by saying. Time out. Server is gracefully shut down.